examples/internal-load-balancer/files/certificate-private-key.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
+MIIEpAIBAAKCAQEAsfkaGuwy3Kqyy9QJpB+9hxL8iDefhduueSBA4bV1Gp0Nngam
+df4xzwZZNzaxVK/8HRAwNsV1so/Ba5rUcTcpSFTI/iy27A6FYl9hYRupmIwsRq4r
+i7Wj4u0ah4XGQbaTqGi16OheLekbN4g8THKG1WngQPnzlsGr20X2UtqQtzooIV3L
+VMGIbS/qZbeRurFdEVHYrQ7fcu6QVzjb2yx5I9IHRwLRm80XbLmeQ8asChcmHzhr
+Q6Yu2CO2IdA7eyqzqpZSX6XYL5RwUmzF3gssFCML4oq4T3OooJcycvEFsfzM29eE
+slMlEjcUSLwsW13s5mhjyDP7t687cqkbU9RcYQIDAQABAoIBAEMBSCTOvK97zOkU
+obSbBp4uNcmLN5IMDXoo/IR97uLgMlpGorFVzKbm+vvgqY762u6OQ7i7Sv65RNcg
+ac9eNzq2QR4A87WYGfvZcS5AXe/DBoBt3pMxBczP++b+JJBlDt6e3a4cniim1Mpp
+jlox3DXJMNcFxAIEVJGvm3fCNi3lJKexnXLX2Xck14u89RTgLzVxdvWuyOey1Jpf
+QoHU0pvu0bVHsygpP6RGYw+teCWPtOCrH6+3/+DlBJt+cG4s0teqkKUcV5SCGjqX
+pdSGWwT9h4ro+h77JyJ9T2rDG7CSa9/rKuI01ylfp1Dz13C8NzEV2Eu7hhAVELx7
+7JSxi6kCgYEA2f5R+nszpzyr5aR08yBmFUJdSzlMbtHVE7aOmd6scYL8nPacQlFZ
+K+8LOQnz3VTWrqZdTUoeFWBeBzXG82ypXwkWCrnErOVUoeSSFHZVBGRiGo9ABOgO
+ws9w1UTz+C3laycbAN6O2hGmsi61hOOGfnxO//r/oMihBrXl8ql77/sCgYEA0QCO
+Te365miKAiJjdY9NylquyO6lrG1u70MaIsQgsnBy/7zurdZOvKQcbAK5o4k9yQns
+W23iKUVBhyiYqzej/ogWeCqMb/Hlg04GLozVGVAruDxmBMRygke2z06fq6OG/pOw
+zaS/sWIlQvLg3P+oXqa4vXLRG3Y73ps4+DolSlMCgYEAzuPu+/InwMX1QvrFP+X0
+3KJilr0nzFl7h70QM/rA4O53P6VSAiLgpbC2jwjiGKKnKZSdIniAers+IkuoqWjs
+ilS+RamvfoQcwZWECsWPxJ5c+WYRVVbEg6GLLe3GOjmc7ghsOXQAUfjEo4FvJ1Ml
+VzA1bf7S0871/L9wINE33jcCgYAftKvh2pZTWcyF4l2gOesdJYCabsmnTLqlhVJz
+r7keGLz5WtoCeR5cs7C0Gf27d1huJRk8DYFfrWfxbxEp+DjiphZyiFHhsZqVvsJl
+UWwKeQUI3/DCDGMqrGJpNI44ZYhXGmSZ7cI+0JrCe0HthMMwCK7B3qcmOVt/u6B6
+16dMjQKBgQCsHPkgWhhoaFztN4EinFSl1Wxu3JHxsVYTTe6Wd4xxkYjydUtdJ2Ht
+GoBIzjewhdl3e2eSWn/zj1V9nNd9xwdoVU8OyofkmaDDbBTe6pv0A2tMAUUwsHfP
+riRa8Et60NzwhbZbQIab5Ii2vq46C+ESK0sx6JrCDG5uVEt5aTJ3DQ==
+-----END RSA PRIVATE KEY-----
```

Comment:
is there a security concern on publishing the priv key for this cert?

Reply:
I included the certificate and related files in the example for convenience since they are deployed to an internal load balancer, but perhaps it makes more sense to just include the certificate creation instructions in the Usage section. What do you think?

Reply:
Fixed by https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/73/commits/fba16bc1d0358f0c82a9598e178e962d244104ec.